page-eight-two.php

```diff
@@ -0,0 +1,225 @@
+		</div>
+		ownCloud is <a href="http://karlitschek.de/2015/06/building-a-platform/">meant to be a platform</a>. This release introduces many improvements to the API, both to make more functionality available for app developers and to improve the existing interfaces.
+                        
+                <div class="row featurerow">
+			<div class="col-md-4 featureblock">
+				<img class="thumbnail" alt="external_storage" src="<?php bloginfo('template_directory'); ?>/assets/img/features/external.png" />
+				<h3 class="featuretitle">Modular Authentication Mechanism Support</h3></br>
+				<p class="featuredesc">External storage backends are now built upon a 'BackendService' class, decoupling authentication from data handling. This reduces complexity in each external storage backend and makes writing new backends easier. It also makes the existing backends more flexible and capable of dealing with a wider range of authentication mechanisms than just 'username-password'.</p>
+			</div>
+
+			<div class="col-md-4 featureblock">
+				<img class="thumbnail" alt="ldap" src="<?php bloginfo('template_directory'); ?>/assets/img/features/avatar.png" />
+				<h3 class="featuretitle">Capabilities and Query Building</h3></br>
+				<p class="featuredesc">A major change is the addition of a capabilities manager which allow apps to expose capabilities in a more generic way. App authors now also have access to a public API for the doctrine query builder.</p>
+			</div>
+			<div class="col-md-4 featureblock">
+				<img class="thumbnail" alt="encryption" src="<?php bloginfo('template_directory'); ?>/assets/img/features/encryption.png" />
+				<h3 class="featuretitle">New and Expanded Functions for Apps</h3></br>
+				<p class="featuredesc">As always, there have been many changes to the ownCloud API. For example, applications can now send notifications to end users and mimetype.js makes it possible to request a mimetype icon from JS. To further aid developers, the command 'occ app-check-code'  will notify about deprecated methods and classes.</p>
+			</div>
+		</div>
+
+               <div class="row featurerow">
+			<div class="col-md-4 featureblock">
+				<img class="thumbnail" alt="external_storage" src="<?php bloginfo('template_directory'); ?>/assets/img/features/external.png" />
+				<h3 class="featuretitle">Improved External API</h3></br>
+				<p class="featuredesc">The OCS Share API now returns a URL when fetching shares, allows setting expire date on new shares directly and returns permissions and expire date on creation.</p>
+			</div>
+                </div>
+
+		<p>See all features of ownCloud on the <a href="/features">Features page</a>.</p>
+		<h2>Get involved!</h2>
+		<p><a target="_blank" href="/promote/" class="btn btn-lg btn-default"> <i class="icon-bullhorn"></i> Discuss ownCloud with others</a></p>
+		<p><a target="_blank" href="/contribute/" class="btn btn-lg btn-default"> <i class="icon-code"></i> join the ownCloud community</a></p>
```

Comment:
This is standard code on many pages that I noticed the other day. I wonder why "join" is with a small "j"?
"Discuss" starts with a capital letter (above) and "ownCloud" is the correct way it should be written below.
But changing "join" to "Join" is quite a global thing in lots of files.